spec/support/shared_examples.rb

```diff
@@ -0,0 +1,8 @@
+shared_examples 'requires sign in' do
+  it 'redirects to sign in page' do
+    logout_user
+    action
+    response.should redirect_to sign_in_path
```

Comment:
You are using `expect(...).to` syntax everywhere else. So `should` just does not feel right -... Read more [here](http://www.betterspecs.org/#expect)